app/base/models.py

```diff
@@ -23,20 +23,30 @@ class User(db.Model, UserMixin):
     def json(self):
```

Comment:
perhaps naming this `to_json` would  be more ideal?

Reply:
I've just removed it  as `user_schema.dump()` can do the same thing